### PR TITLE
Arcbox dashboards

### PR DESCRIPTION
**Pull request by dkirby-ms**
Partially addresses #718

### CODE REVIEW

File: azure_jumpstart_arcbox/artifacts/Bootstrap.ps1 [hunk 1]
```diff
@@ -178,6 +181,11 @@ $Trigger = New-ScheduledTaskTrigger -AtLogOn
 $Action = New-ScheduledTaskAction -Execute "PowerShell.exe" -Argument 'C:\ArcBox\DataServicesLogonScript.ps1'
 Register-ScheduledTask -TaskName "DataServicesLogonScript" -Trigger $Trigger -User $adminUsername -Action $Action -RunLevel "Highest" -Force
 
+# Creating scheduled task for DataServicesLogonScript.ps1
```
Comment (likamrat) [inline azure_jumpstart_arcbox/artifacts/Bootstrap.ps1:184]:
```suggestion
# Creating scheduled task for MonitorWorkbookLogonScript.ps1
```

File: azure_jumpstart_arcbox/azuredeploy.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
     }
   },
   "variables": {
-    "templateBaseUrl": "https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_jumpstart_arcbox/",
+    "templateBaseUrl": "https://raw.githubusercontent.com/microsoft/azure_arc/arcbox_dashboards/azure_jumpstart_arcbox/",
```
Comment (likamrat) [inline azure_jumpstart_arcbox/azuredeploy.json:57]:
```suggestion
    "templateBaseUrl": "https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_jumpstart_arcbox/",
```

File: docs/azure_jumpstart_arcbox/_index.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ ArcBox deploys one single-node Rancher K3s cluster (_ArcBox-CAPI-MGMT_), which i
 
 ### Hybrid Unified Operations
 
-ArcBox deploys several management and operations services that work with ArcBox's Azure Arc resources. These resources include an an Azure Automation account, an Azure Log Analytics workspace with the Update Management solution, Azure Policy assignments for deploying Log Analytics agents on Windows and Linux, Azure Policy assignment for adding tags to resources, and a storage account used for staging resources needed for the deployment automation.
+ArcBox deploys several management and operations services that work with ArcBox's Azure Arc resources. These resources include an an Azure Automation account, an Azure Log Analytics workspace with the Update Management solution, an Azure Monitor workbook, Azure Policy assignments for deploying Log Analytics agents on Windows and Linux, Azure Policy assignment for adding tags to resources, and a storage account used for staging resources needed for the deployment automation.
```
Comment (likamrat) [inline docs/azure_jumpstart_arcbox/_index.md:44]:
```suggestion
ArcBox deploys several management and operations services that work with ArcBox's Azure Arc resources. These resources include an an Azure Automation account, an Azure Log Analytics workspace with the Update Management solution, an Azure Monitor workbook, Azure Policy assignments for deploying Log Analytics agents on Windows and Linux Azure Arc-enabled servers, Azure Policy assignment for adding tags to resources, and a storage account used for staging resources needed for the deployment automation.
```

File: docs/azure_jumpstart_arcbox/_index.md [hunk 4]
```diff
@@ -227,8 +228,14 @@ After deployment is complete, its time to start exploring ArcBox. Most interacti
 
   ![Screenshot showing Azure Data Studio usage](./azdatastudio.png)
 
+* Open the [ArcBox Azure Monitor workbook](./workbook/_index.md) and explore the visualizations and reports of hybrid cloud resources. A [dedicated README](./workbook/_index.md) is available with more detail on usage of the workbook.
```
Comment (likamrat) [inline docs/azure_jumpstart_arcbox/_index.md:231]: Need to change these links to the Jumpstart full path

**Review by likamrat — CHANGES_REQUESTED** [review]
- Minor edits
 - Link update

File: docs/azure_jumpstart_arcbox/_index.md (hunk 4 above)
Comment (dkirby-ms) [reply docs/azure_jumpstart_arcbox/_index.md:231]: by design

**Review by likamrat — APPROVED** [review]
LGTM. Great work!

**Pull request merged**